sdk/python/foundation-models/llama2/promptFlow/Chat_flow_sample/chatting_with_Gitdoc/get_url.py

```diff
@@ -0,0 +1,6 @@
+from promptflow import tool
+
+@tool
+def get_url(input: str) -> str:
+  """Returns the input URL."""
+  return input
```

Comment:
This tool looks it does nothing

Reply:
I have removed get_url.py file and passed URL directly through 'chatting_with_user' node.